docs/end-user/policies/envbinding.md

```diff
@@ -1,3 +1,118 @@
 ---
-title: Env-Binding
----
\ No newline at end of file
+title: Multi-Environment Deployment
+---
+
+This documentation will introduce how to use env-binding to automate multi-stage application rollout across multiple environments.
+
+## Background
+
+Users usually have two or more environments to deploy applications to. For example, dev environment to test the application code, and production environment to deploy applications to serve live traffic. For different environments, the deployment configuration also has some nuance.
+
+## Multi-env Application Deployment
+
+Deploy the following Application:
+
+```yaml
+# app.yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: policy-demo
+  namespace: default
+spec:
+  components:
+    - name: nginx-server
+      type: webservice
+      properties:
+        image: nginx:1.21
+        port: 80
+
+  policies:
+    - name: demo-policy
+      type: env-binding
+      properties:
+        clusterManagementEngine: single-cluster
+        envs:
+          - name: test
+            patch:
+              components:
+                - name: nginx-server
+                  type: webservice
+                  properties:
+                    image: nginx:1.20
+                    port: 80
+            placement:
+              namespaceSelector:
+                name: test
+  workflow:
+    steps:
+      - name: deploy-test-env
+        type: multi-env
+        properties:
+          # Specify the policy name
+          policy: demo-policy
+          # Specify the env name in the policy
+          env: test
+```
+
+This will deploy the Application `policy-demo` to the namespace `test` (test environment), and patch the parameters in the component `nginx-server` to adapt to the test environment.
+
+We apply the Application `policy-demo` in the example.
+
+```shell
+kubectl apply -f app.yaml
+```
```

Comment:
You have already told the end-users to deploy the application in Line https://github.com/oam-dev/kubevela.io/pull/240/files#diff-976f739e1afc2db7906a94dbc2c0f84d9175c5d7347fa921b3588126911f54bdR13.